nh_ews/notifications.py

```diff
@@ -56,7 +56,7 @@ class nh_clinical_notification_medical_team(orm.Model):
     _name = 'nh.clinical.notification.medical_team'
     _inherit = ['nh.clinical.notification']
     _description = 'Inform Medical Team?'
-    _notifications = [{'model': 'doctor_assessment', 'groups': ['nurse']}]
+    _notifications = [{'model': 'doctor_assessment', 'groups': ['doctor']}]
```

Comment:
Groups in notifications are the user group to trigger the notification for so as the task 'Assessment Required' is triggered for doctors on an inform medical team task being completed it should have been doctor and not nurse.